src/eq_schema/schema/Section/index.js

```diff
@@ -91,12 +96,17 @@ class Section {
 
     this.groups = [new Group({ ...section, pages }, ctx)];
 
-    if (section.introductionTitle && section.introductionContent) {
+    if (
+      section.introductionEnabled &&
+      section.introductionTitle &&
+      section.introductionContent
+    ) {
```

Comment:
SHouldn't this also check for the presence of section.introductionPageDescription, otherwise if could be trying to use a value it doesn't have?